packages/css-framework/src/objects/_wrapper.scss

```diff
@@ -1 +1,9 @@
-// Wrapper Object
\ No newline at end of file
+.container {
+  max-width: 1440px;
+  padding: 0 spacing(4);
+  margin: 0 auto;
+  @media only screen and (min-width: 768px) {
```

Comment:
Isn't there a mixing or something for this?

Reply:
It doesn't currently work. @hxltrhuxze is planning to recreate it soon.

Reply:
hmm. ok, though I've used it for Nav. What doesn't work? Maybe leave a Todo in here so we don't forget